plugins/gatsby-source-jenkinsplugins/utils.js

```diff
@@ -75,7 +75,7 @@ const getPluginContent = async ({wiki, pluginName, reporter, createNode, createC
             id: `${pluginName} <<< JenkinsPluginWiki`,
             name: pluginName,
             url: url,
-            baseHref: `${path.dirname(url)}/`,
+            baseHref: `${url && path.dirname(url)}/`,
```

Comment:
won't this output `false/`?
`path.dirname(url || '')` would work maybe?

Honestly if there's no url, then there probalby isn't need to generate a wiki object.

Reply:
> there probalby isn't need to generate a wiki object

yep, then the error message can be created in UI and we don't need to download it from API. I tried that in a new commit.

Reply:
I'm going to revert this line, there's no need for it, and it could result in a random false.